src/anomalib/models/image/supersimplenet/anomaly_generator.py

```diff
@@ -0,0 +1,142 @@
+"""Anomaly generator for the SuperSimplenet model implementation."""
+
+# Copyright (C) 2024 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import torch
+import torch.nn.functional as F  # noqa: N812
+from torch import nn
+
+from anomalib.data.utils.generators import generate_perlin_noise
+
+
+class SSNAnomalyGenerator(nn.Module):
```

Comment:
Not sure about the naming here. Why not simply use `AnomalyGenerator`? You don't use the `SSN` prefix for the other submodules.

Reply:
This is now just `AnomalyGenerator`.